### DISCUSSION TITLE
Images for ticks

### DISCUSSION THREAD

**Discussion by slochower** [bokeh/bokeh#14135] [General]
Is it possible to use `CustomJSTickFormatter` to return an image (or something other than a string) to format tick labels? For categorical plots, it can be very helpful to have images as the tick label, instead of a string.

**Comment by bryevdv** [comment]
No one has ever asked about this before. It's not currently possible to have anything other than text tick labels.

**Comment by slochower** [comment]
Ah, okay. Consider it a feature request then :) There are a few examples of similar things floating around for `matplotlib`: https://stackoverflow.com/questions/77732494/add-image-below-x-axis-in-matplotlib and https://stackoverflow.com/questions/69538086/how-to-use-images-as-xtick-labels-in-seaborn-matplotlib-barplot...

**Comment by bryevdv** [comment]
@slochower If you want to request this as a feature, please open an issue: https://github.com/bokeh/bokeh/issues
